packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/stop-bot-modal.tsx

```diff
@@ -91,52 +93,49 @@ const StopBotModalContent = ({
 
 const StopBotModal = ({
     is_running,
-    is_dialog_open,
     is_contract_dialog_open,
     is_stop_bot_dialog_open,
     is_multiplier,
     closeMultiplierContract,
     stopMyBot,
     toggleStopBotDialog,
     loadDataStrategy,
```

Comment:
We can remove loadDataStrategy @maryia-matskevich-deriv

Reply:
yes, and a lot of different things should be fixed as types